src/SchoolSubjectsList.tsx

```diff
@@ -43,7 +42,7 @@ const SchoolSubjectsList: React.FC<Props> = ({ schoolCode, year, season }) => {
           .sort((a, b) => a[0].localeCompare(b[0]))
           .map(([code, name]) => (
             <Link key={code} to={`/${year}/${season}/${schoolCode}/${code}`}>
-              {getOrKey(name, weirdSubjectNames)}
+            {getOrKey(name.name, weirdSubjectNames)}
```

Comment:
perhaps we should rename `name`? `name.name` is a little odd

Reply:
Thanks. I changed it. I don't think weird subject name is necessary anymore because the API has cleaned up the data already. Should we remove this?

Reply:
Eh some of the subject names are weird simply because Albert has them written weirdly like "Child/Adoles Mental Hlth Stds"